central/views/imagecve/view_impl.go

```diff
@@ -103,8 +103,24 @@ func (v *imageCVECoreViewImpl) Get(ctx context.Context, q *v1.Query, options vie
 		return nil, err
 	}
 
+	var cveIDsToFilter []string
+	if q.GetPagination().GetLimit() > 0 || q.GetPagination().GetOffset() > 0 {
+		// TODO(@charmik) : Update the SQL query generator to not include 'ORDER BY' and 'GROUP BY' fields in the select clause (before where).
```

Comment:
Out of curiosity, what did the original SQL look like?

Reply:
```
select jsonb_agg(distinct(image_cves.Id)) as cve_id, image_cves.CveBaseInfo_Cve as cve, max(image_cves.Cvss) as cvss_max from image_cves group by image_cves.CveBaseInfo_Cve order by max(image_cves.Cvss) desc, image_cves.CveBaseInfo_Cve asc LIMIT 20
```

Currently SQL looks like this. But the `v1.Query` only asks to select `cve_ids` and has `cve` in group by clause and `cvss_max` in order by. So it doesn't need to select `cve` and `cvss_max` here.